testsuite/forge-cli/src/main.rs

```diff
@@ -452,9 +452,13 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
             })),
         "config" => config.with_network_tests(vec![&ReconfigurationTest]),
         "network_partition" => config.with_network_tests(vec![&NetworkPartitionTest]),
-        "network_latency" => config
-            .with_network_tests(vec![&NetworkLatencyTest])
-            .with_success_criteria(SuccessCriteria::new(4000, 10000, true, None, None)),
+        "three_region_simulation" => config
+            .with_initial_validator_count(NonZeroUsize::new(20).unwrap())
+            .with_initial_fullnode_count(10)
```

Comment:
I don't think we need so many full nodes, 5 should be sufficient. I will take a look at other tests to optimize the number of FNs but let's try lower number for this test?